### ISSUE TITLE
Specified license file does not exist for Quartus Prime Standard

### ISSUE THREAD

**Issue by developandplay** [intel/FPGA-Devcloud#38]
I am trying to run the Quartus Prime Standard edition on one of the compile servers.

Steps I took:
1. Use `tools_setup` to specify Quartus Prime Standard.
```
$ tools_setup 

Which tool would you like to source?
1) Quartus Prime Lite
2) Quartus Prime Standard
3) Quartus Prime Pro
4) HLS
5) Arria 10 PAC Compilation and Programming - RTL AFU, OpenCL
6) Arria 10 - OneAPI, OpenVINO
7) Stratix 10 PAC Compilation and Programming - RTL AFU, OpenCL
8) Stratix 10 - OneAPI, OpenVINO

Number: 2
sourcing /glob/development-tools/versions/intelFPGA/18.1/init_quartus.sh
```
2. Overwrite `$QUARTUS_ROOTDIR_OVERRIDE` which is still set to Quartus Pro.
```
$ echo $QUARTUS_ROOTDIR_OVERRIDE
/glob/development-tools/versions/oneapi/2022.1.2/oneapi/intelfpgadpcpp/2022.1.0/QuartusPrimePro/19.2/quartus
$ export QUARTUS_ROOTDIR_OVERRIDE=/glob/development-tools/versions/intelFPGA/18.1/quartus
```
3. Run `quartus_map` in my project
```
$ quartus_map otma_bringup
Error (292026): Specified license file does not exist.
```
4. Trying a different license file also leads to an error
```
$ echo $LM_LICENSE_FILE
/usr/local/licenseserver/quartus.lic
$ export LM_LICENSE_FILE=/usr/local/licenseserver/psxe.lic
$ quartus_map otma_bringup
Error (292027): Specified license does not contain information required to run the Quartus Prime software.
```

I'm assuming the `psxe.lic` is not the proper license file for Quartus Prime but what is the location of the correct license file?

**Comment by developandplay** [comment]
I guess I was using the wrong server if I use the `Stratix 10 PAC Compilation and Programming - RTL AFU, OpenCL` then the license file is available.

However now the tunneling is not working as it is supposed to for the GUI (u123456 of course replaced with my own user):
```
ssh -L 4002:s005-n005:22 devcloud
ssh u123456@localhost -p 4002
The authenticity of host '[localhost]:4002 ([127.0.0.1]:4002)' can't be established.
ECDSA key fingerprint is SHA256:xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx.
Are you sure you want to continue connecting (yes/no/[fingerprint])? yes
Warning: Permanently added '[localhost]:4002' (ECDSA) to the list of known hosts.
u123456@localhost: Permission denied (publickey,gssapi-keyex,gssapi-with-mic).
```
Or if I'm using `x2goclient` then I get [this](https://github.com/intel/FPGA-Devcloud/issues/12) issue.

**Comment by llandis** [comment]
Glad you got the first license step going. Select A10 or S10 machines. or manually set it 
export LM_LICENSE_FILE=/usr/local/licenseserver/quartus.lic

did you try launching x2go after running the tunneling command. Although it throws an error, x2go should launch.

**Comment by developandplay** [comment]
@llandis 
I'm not sure what was wrong with my original account.
After some time I couldn't even log using the direct ssh connection. (Maybe I got blocked after too many bad connection requests?)
I just created a new account and made some more progress.

Interestingly now I get asked for a password using x2go (the ssh tunneling issue remains as explained above).
However I'm not sure what password I would have to enter and as such get this error:
`Permission denied (publickey, password)`
![image](https://user-images.githubusercontent.com/34752929/155504129-5552baca-5930-4604-9726-b584068bb8a3.png)
